# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsINode.h
# Commit: 29195ee87cf4
# Full Hash: 29195ee87cf436e0b76854ec86c114c1d1ced8fe
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-05-10 09:29:34
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Bug 1932150 - Allow using flat tree order for point comparing in selection r=jjaschke,smaug,dom-core
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231588
# ==============================================================================

diff -r a86c7713f936 -r 29195ee87cf4 dom/base/nsINode.h
--- a/dom/base/nsINode.h	Fri May 09 17:16:36 2025 +0000
+++ b/dom/base/nsINode.h	Fri May 09 17:16:36 2025 +0000
@@ -1190,7 +1190,7 @@
    *   2. For contents that are slotted into a UA shadow tree, use its
    *   parent rather than the slot element.
    */
-  inline nsIContent* GetFlattenedTreeParentNodeForSelection() const;
+  inline nsINode* GetFlattenedTreeParentNodeForSelection() const;
 
   inline mozilla::dom::Element* GetFlattenedTreeParentElement() const;
   inline mozilla::dom::Element* GetFlattenedTreeParentElementForStyle() const;